cosmos/kaiyo.json

```diff
@@ -120,6 +120,12 @@
       "coinMinimalDenom": "ibc/1B38805B1C75352B28169284F96DF56BDEBD9E8FAC005BDCC8CF0378C82AA8E7",
       "coinDecimals": 18,
       "coinGeckoId": "ethereum"
+    },
+    {
+      "coinDenom": "bKUJI",
+      "coinMinimalDenom": "factory/kujira15e8q5wzlk5k38gjxlhse3vu6vqnafysncx2ltexd6y9gx50vuj2qpt7dgv/boneKuji",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kaiyo/factory/kujira15e8q5wzlk5k38gjxlhse3vu6vqnafysncx2ltexd6y9gx50vuj2qpt7dgv/bKUJI.png"
```

Comment:
@tvl83 Do you want to use `bKUJI` as fee currency?

Reply:
no. did i put it in the wrong location in the json?  -- i found the mistake ill correct it with a commit.